courses/management/commands/manage_program_certificates.py

```diff
@@ -0,0 +1,144 @@
+        )
+        parser.add_argument(
+            "--revoke", dest="revoke", action="store_true", required=False
+        )
+        parser.add_argument(
+            "--unrevoke", dest="unrevoke", action="store_true", required=False
+        )
+        parser.add_argument(
+            "--create", dest="create", action="store_true", required=False
+        )
+
+        super().add_arguments(parser)
+
+    def handle(self, *args, **options):  # pylint: disable=too-many-locals
+        """Handle command execution"""
+
+        revoke = options.get("revoke")
+        unrevoke = options.get("unrevoke")
+        create = options.get("create")
+        program = options.get("program")
+
+        if not (revoke or unrevoke) and not create:
+            raise CommandError(
+                "The command needs a valid action e.g. --revoke, --unrevoke, --create."
+            )
+        try:
+            user = fetch_user(options["user"]) if options["user"] else None
+        except User.DoesNotExist:
+            user = None
+
+        # A program is needed for revoke/un-revoke and certificate creation
+        if not program:
+            raise CommandError("The command needs a valid program.")
+
+        # Unable to obtain a program object based on the provided courseware id
+        try:
+            program = Program.objects.get(readable_id=program)
+        except Program.DoesNotExist:
+            raise CommandError(
+                "Could not find program with readable_id={}.".format(program)
+            )
+
+        # Handle revoke/un-revoke of a certificate
+        if revoke or unrevoke:
+            if not user:
+                raise CommandError("Revoke/Un-revoke operation needs a valid user.")
+
+            revoke_status = manage_program_certificate_access(
+                user=user,
+                program=program,
+                revoke_state=True if revoke else False,
+            )
+
+            if revoke_status:
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        "Certificate for {} has been {}".format(
+                            "program: {}".format(program),
+                            "revoked." if revoke else "un-revoked.",
+                        )
+                    )
+                )
+            else:
+                self.stdout.write(self.style.WARNING("No changes made."))
+
+        # Handle the creation of the certificates.
+        # Also check if the certificate creation was requested with grade override. (Generally useful when we want to
+        # create a certificate for a user while overriding the grade value)
+        elif create:
+
+            # While overriding grade we force create the certificate
+            certificate, created_cert = generate_program_certificate(
+                user=user,
+                program=program,
+            )
+
+            if certificate and created_cert:
```

Comment:
if `created_cert` is True this already tells us that a certificate was created, I don't think we still need to check for the `certificate` object.

```suggestion
            if created_cert:
```